### ISSUE TITLE
Output File Is Resized

### ISSUE THREAD

**Issue by spot92** [lengstrom/fast-style-transfer#217]
The code ran successfully on a file sized 234 MB, but the output was only sized 5.21 MB. Why?

**Comment by SourceCode** [comment]
Yes - noticed the same thing and looking into it myself, have not found the cause or a solution, but have the same issue.  Encoded video results are awful.

**Comment by SourceCode** [comment]
So basically this is in evaluate.py, there is a function, ffwd_video - if you adjulst the default parameters from preset="medium", bitrate="2000k" to preset="slow", bitrate="20000k", you'll notice you're now getting the quality you expect.
